examples/delete_door_code.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""Example script."""
+
+# Parse Arguments
+# Import project path
+import argparse, time
+import os
+import sys
+
+# Import pyvera
+from pyvera import VeraController, VeraLock
+
+
+def main() -> None:
+    """Run main code entrypoint."""
+    sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
+
+    parser = argparse.ArgumentParser(description="set-and-delete-door-code")
+    parser.add_argument(
+        "-u", "--url", help="Vera URL, e.g. http://192.168.1.161:3480; Name eg: \"John Doe\", Pin eg: \"12345678\"", required=True
+    )
+    parser.add_argument(
+        "-s", "--slot", help="Slot eg: \"4\"", required=True
+    )
+    args = parser.parse_args()
+
+    # Start the controller
+    controller = VeraController(args.url)
+    controller.start()
+
+    try:
+        # Get a list of all the devices on the vera controller
+        all_devices = controller.get_devices()
```

Comment:
`get_devices` takes a second parameter to filter the device type - which might be slightly more elegant.

Reply:
I was following the existing code flow, minimizing no:of breaking changes

Reply:
I think you still need to make this change.

Reply:
Done